python/graphscope/interactive/query.py

```diff
@@ -141,6 +147,27 @@ def execute(self, query):
             raise RuntimeError("Interactive query is closed.")
         return self._client.submit(query)
 
+    def traversal_source(self):
+        """Create a GraphTraversalSource and return.
+        Once "g" has been created using a connection, we can start to write
+        Gremlin traversals to query the remote graph.
+
+        Examples:
+
+            ..code:: python
+
+                sess = graphscope.session()
+                graph = load_modern_graph(sess, modern_graph_data_dir)
+                interactive = sess.gremlin(graph)
+                g = interactive.traversal_source()
+                print(g.V().both()[1:3].toList())
+                print(g.V().both().name.toList())
+
+        Returns:
+            `GraphTraversalSource`
+        """
+        return traversal().withRemote(DriverRemoteConnection(self._graph_url, "g"))
```

Comment:
Is the parameter `"g"` a hard-coded constant value?

Reply:
AFAIK yes.

Reply:
Then could `"g"` will be used repeatly?

Reply:
Yes if you are talking about using `g` repeatedly across graph instances.

Reply:
What will happen on the same graph instance?

Reply:
Nothing happens, you can get two traversal source that works normally.
They are same if you look at the initialization of `Client` in https://github.com/alibaba/GraphScope/blob/dd2a58ac8ddd1600a997ef0312895e5f87768168/python/graphscope/interactive/query.py#L46